skipruntime-ts/core/src/skipruntime_utils.ts

```diff
@@ -36,3 +36,14 @@ export class Max implements Accumulator<number, number> {
     return value < acc ? acc : null;
   }
 }
+
+export function getReactiveResponse(
+  httpHeaders: Headers,
```

Comment:
Suggestion: how about we name this `parseReactiveResponse()` and have it take the header value as parameter? It adds a small burden to the caller (who would have to call `parseReactiveResponse(headers.get("Skip-Reactive-Response"))` instead of `getReactiveResponse(headers)` but it would not limit the scope to HTTP `Headers` objects.

Reply:
I applied this renaming but, since we're in dynamic language land already, I just made overloaded the function on both input types so callers don't need to grab the header themselves.  Saves a few keystrokes, but more importantly IMO minimizes the number of places where we have that string literal.

Reply:
Orthogonal: We should probably define a global constant for the header name in any case.